index.markdown

```diff
@@ -19,8 +23,23 @@ installation:
   - command: "conda install rustpython -c conda-forge"
 
 goals:
-  - goal: "Full Python-3 environment entirely in Rust (not CPython bindings) with a clean implementation, without compatiblity hacks. "
-  - goal: "Fast, reliable and secure implementation of Python that can be used with Rust or compiled to WebAssembly. "
+  - goal:
+      Full Python-3 environment entirely in Rust (not CPython bindings), with
+      a clean implementation and no compatiblity hacks.
+    # TODO: integrate this into the "goals" boxes
+    progress:
```

Comment:
I like the progress boxes or text. Is the progress box supposed to show somewhere on the homepage?

Reply:
I was thinking something like split the goals boxes into 2 vertically with the goal on top and progress on bottom:

![image](https://user-images.githubusercontent.com/33094578/101806469-9e1dcf00-3ad9-11eb-86dd-ad33c69aa370.png)

(very rough sketch, of course)

Reply:
yeah that works :D 

If you want, you don't have to stick to the background I used. You can probably use borders or some form of highlight.